code/src/index.css

```diff
@@ -1,13 +1,133 @@
+* {
+  margin: 0; 
+  padding: 0;  
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: content-box;
+  -webkit-font-smoothing: antialiased; 
+  text-rendering: optimizeLegibility;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
+  font-family: -apple-system, BlinkMacSystemFont, Helvetica;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #181818;
+  color: #fff;
+}
+
+.container {
+  margin: 2rem;
+}
+
+a {
+  text-decoration: none;
+  font-size: 14px;
+  line-height: 14px;
+}
+
+.artist-name {
+  color: #a0a0a0;
+  font-size: 14px;
+}
+
+.album-name a {
+  font-weight: 700;
+  color: #ffffff;
+}
+
+.album-name {
+  margin: 10px 0;
+}
+
+.album {
+  display: flex;
+  flex-direction: column;
+  max-width: 70vw;
+  padding: 15px;
+}
+
+.flex-container {
+  margin: 2rem auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
```

Comment:
I think you could have used gap here instead of putting padding around the .album!